shell/protoc.sh

```diff
@@ -29,9 +29,24 @@ if [[ -z $uid ]] || [[ -z $gid ]]; then
   gid=1000
 fi
 
+get_import_basename() {
+  basename $(go list -f '{{ .Path }}' -m "$1" | sed 's|/v[0-9]||') # sed removes the version off the module path if present
+}
+for import in $(get_list "go-protoc-imports"); do
+  go install "$import"
```

Comment:
Repeating an earlier comment, can we use `gobin` here? Downside is we'd need to manually pass the executable path to protoc

Reply:
I briefly looked at this, I might be misunderstanding `gobin`s use here. It looked like our script was for executing binaries. The purpose of this install isn't to execute anything, but rather to get the dependency pulled into the modules cache.

Reply:
Right, but `go install` doesn't work great on <1.16, so we've used gobin to do this for the time being.

Reply:
Added a version check and tested with
```
$ go version | awk '{ print $3 }' | sed 's|go||'
1.16.6
$ /home/zaq/projects/outreach/settings/scripts/shell-wrapper.sh protoc.sh
go install github.com/getoutreach/schema@latest: module github.com/getoutreach/schema@latest found (v0.0.0-20210830182546-82ce20ed15eb), but does not contain package github.com/getoutreach/schema
/home/zaq/.asdf/installs/golang/1.16.6/packages/pkg/mod/github.com/getoutreach/schema@v0.0.0-20210830182546-82ce20ed15eb
 :: Generating GRPC Clients
^C
zaq@vm:~/projects/outreach/settings (schema_import_test)
$ vim .tool-versions
zaq@vm:~/projects/outreach/settings (schema_import_test)
$ /home/zaq/projects/outreach/settings/scripts/shell-wrapper.sh protoc.sh
Go version must be greater than 1.16.0 to use 'go-protoc-imports' feature
zaq@vm:~/projects/outreach/settings (schema_import_test)
$ go version
go version go1.15.2 linux/amd64

```